internal/pkg/service/appsproxy/proxy/apphandler/authproxy/pagewriter.go

```diff
@@ -70,11 +71,11 @@ func (pw *pageWriter) WriteErrorPage(w http.ResponseWriter, req *http.Request, o
 		}
 	}
 
-	pw.pageWriter.WriteErrorPage(w, req, opts.Status, messages, "", opts.RequestID)
+	pw.pageWriter.WriteErrorPage(w, req, &pw.app, opts.Status, messages, "", opts.RequestID)
 }
 
 func (pw *pageWriter) ProxyErrorHandler(w http.ResponseWriter, req *http.Request, err error) {
-	pw.pageWriter.ProxyErrorHandler(w, req, err)
+	pw.pageWriter.ProxyErrorHandler(w, req, pw.app, err)
```

Comment:
No, because there are error that are not related to an app (appID missing for example), but app is always known, when we creating Go reverse proxy, that uses `ProxyErrorHandler`.

Reply:
It is reflected also in the templates:
![image](https://github.com/keboola/keboola-as-code/assets/19371734/f1c6e498-9cf3-44b3-b73e-94cb45aaa2b3)